### PR TITLE
Updated out of date instructions

### PR DESCRIPTION
**Pull request by mzrpdx**
Worked through tutorial and updated things that were unclear or out of date.

### CODE REVIEW

File: advanced/external_references/README.md [hunk 1]
```diff
@@ -259,6 +259,17 @@ end
 Add a new input parameter and a new output parameter *color* as shown above. In the *run()*
 function, just assign the input color to the output color.
 
+In order for the LUA script and LUA interface to work together, you need to add a line for the
+parameter *color* in the interface, as well.
+
+```
+function interface(INOUT)
```
Comment (smirko-dev) [inline advanced/external_references/README.md:266]:
```suggestion
function interface(IN, OUT)
```

File: advanced/external_references/README.md [hunk 2]
```diff
@@ -170,8 +170,8 @@ We are still missing the mesh *RoadPlane*. Copy it from the Project Browser and
 Now we build our scene using the imported resources. Add a Node and a MeshNode as its child.
 Assign the imported material and mesh to the MeshNode. You should see a distorted piece of
 road in the Preview. Adjust the scaling as shown in the picture to match the scale of the
-other objects we are going to import shortly. Finally, rotate the Node -90° on the X-axis,
-this should put the road flat on the ground where it belongs.
+other objects we are going to import shortly. In order to see the street correctly in the 
+Preview you might want to move the camera a bit up and rotate it downwards.
```
Comment (violinyanev) [inline advanced/external_references/README.md:174]: why make it more vague? "rotate a bit up" is totally ambigous in a 3D scene

File: advanced/external_references/README.md (hunk 1 above)
Comment (violinyanev) [reply advanced/external_references/README.md:266]: please ignore, your version is the correct one

**Review by violinyanev — CHANGES_REQUESTED** [review]

File: advanced/external_references/README.md (hunk 2 above)
Comment (mzrpdx) [reply advanced/external_references/README.md]: You are right, we made it more clear and fixed it in the latest commit.

**Pull request merged**